python/static/resources/corpora/wordnet/LICENSE

```diff
@@ -0,0 +1,31 @@
+WordNet Release 3.0
```

Comment:
Do we still need it or we will upload it the lib as a binary file?

Reply:
We just need the `.kv` file generated from this corpus. I will delete unecessary files.

Reply:
No longer needed, all we need are the `.kv` files. I will remove all other files.

Reply:
Yes, this is part of `WordNet` which is used in the Lemmatizer.